src/coreclr/System.Private.CoreLib/src/System/Reflection/Emit/EnumBuilder.cs

```diff
@@ -323,11 +323,13 @@ public override Type MakeByRefType()
             return SymbolType.FormCompoundType("&", this, 0)!;
         }
 
+        [System.Diagnostics.CodeAnalysis.RequiresDynamicCodeAttribute("The native code for the array might not be available at runtime.")]
```

Comment:
Could you please change this message everywhere according to the feedback in https://github.com/dotnet/runtime/pull/66025#discussion_r818111092 ?

(I am not going to update the messages in my change.)